src/Main.c

```diff
@@ -82,6 +85,7 @@ httpse_getopt_long(int argc, char **argv, HttpseRulesetOptions *roptions)
 		{"tr-encoding",     no_argument,       NULL, 263},
 		{"useragent",       required_argument, NULL, 'A'},
 		{"verbose",         no_argument,       NULL, 'v'},
+		{"min-tls",         required_argument, NULL, 266},
```

Comment:
If we allow an user to specify the minimum TLS versions to be used, the error message would become confusing. IMHO, this require some huge efforts to resolve given the existing code logic. 

https://github.com/cschanaj/xhttpse2/blob/20b20f42986720a67068d5c87d5503b5f640a3bb/src/HttpseCode.c#L22 

In case you decide to proceed with this PR, it is more preferable to use curl compatible commands:

```
 -1, --tlsv1         Use TLSv1.0 or greater
     --tlsv1.0       Use TLSv1.0
     --tlsv1.1       Use TLSv1.1
     --tlsv1.2       Use TLSv1.2
     --tlsv1.3       Use TLSv1.3
```